meetings/2021-12-01.md

```diff
@@ -0,0 +1,124 @@
+* ...: In my opinion the client should not need to understand anything about ACP, just the data grants. The RS should not have to understand data grants because the AS has a way to coordinate with the RS. So in a way the AS would pretty much translate the data grants into ACRs. AS would need to understand both (ACP and data grants). The RS would have data grants and project them as ACRs.
+* Elf: One question here is how Alice is setting the access policies?
+* ...: The way Alice is granting access to Bob, it's Bob's choice what client they use. So Alice gives access to Bob and Bob gives access to a client (delegated data grants, narrowing access). So Bob can delegate read only access to a client using a data grant.
+* ...: Alice uses her authz server. I don't think Alice sets ACRs, Alice uses a UI. Bob uses his authorization agent.
+* Matthieu: How do data grants and verifiable credentials relate to each other?
+* Elf: Possibly linked data capabilities should need to be defined. Really we're trying to respond to the problem of OAuth scopes. The app doesn't know what Alice gave access to for Bob.
+* Matthieu: So the data grants vocabulary is the important part here that allows us to define a scope of which resources and access modes for something.
+* Elf: I think the approach I'm describing here the AS would set the permissions and the users would not use a random client to set permissions. They should use a client that they trust. To understand what the consequence of their decisions is.
+* Emelia: I think there might be different thing here. There exists Access Grants that work through signed JWTs that's a way (Alice creates a grant and Bob exchanges it from the AS for an access token).
+* ...: As far as I recall the only entity that can change ACRs is the owner of the resource. So Alice could allow access to Bob but Bob would not as not owner be able to change the ACR. I don't think any client should change any ACR.
+* Henry: I have another answer which I'll implement in a month or two. I can also show how one can do it in a declarative way. But I've talked about it for about 2 years, so I should implement it to prove the launcher app works. I think the best is to implement both and see what the commonalities are.
+* Elf: I think the important bit is the user having one UI or trusted App to set permissions. One place where they set them and they understand how it works and the consequences. The authz agent is the same sort of thing.
+* Henry: Perhaps one should start with questions. Your main claim is one app to do all authz (but it can do it only on one server).
+* Elf: Alice's authz agent would be aware of all the access granted to and by her.
+* ...: Both a launcher app and authz agent would have similar roles, but instead of living on a device, the agent would be a service in the cloud.
+* Henry: Let's keep in mind the follow your nose approach. For example if access was granted to the friend of a friend.
```

Comment:
```suggestion
* Elf: Alice's authz agent would be aware of all the access granted to and by her.
  * Both a launcher app and authz agent would have similar roles, but instead of living on a device, the agent would be a service in the cloud.
* Henry: Let's keep in mind the follow your nose approach; for example, if access was granted to the friend of a friend.
```